BIT_+_ADR_UK_synth_data.ipynb

```diff
@@ -368,6 +368,8 @@
         "        return \"string\"\n",
         "    # We then check if it's numeric, or predominantly numeric with some exceptions:\n",
         "    elif(check_if_numeric(x) == True): \n",
+        "        if x.min() > 19200101 and x.max() < 20260101:",
```

Comment:
This is a good catch re. a datetime format we were unaware of when writing the code!

Thinking a little about edge cases here: 

- If someone has a numerical column where all the values are between 19200101 and 20260101, then this will class it as a datetime column.  This isn't fatal, because they can redefine it as numerical using the method we provide later in the notebook, but I wonder if we can reduce the risk.  Some thoughts:
  - Screen for values that are impossible if these are date time formats  (eg. looking for digits in the month position outside of the range 0 to 12 and/or digits in the day position outside of the range 0 to 30)

Reply:
I've pushed a commit which does what you describe.

Reply:
Sorry, auth issues, commit now pushed https://github.com/BehaviouralInsights/BIT-ADRUK-synthetic-data-tool/pull/2/commits/28f0c30abb80b649794ec1f56afe7d84f1053248